configs/SPRACINGH7EVO/config.h

```diff
@@ -0,0 +1,285 @@
+#define SPRACING_PIXEL_OSD_PIXEL_GATING_DEBUG_PIN       PB0 // pwm 6
+#define SPRACING_PIXEL_OSD_PIXEL_BLANKING_DEBUG_PIN     PB1 // pwm 5
+
+#define BEEPER_INVERTED
+#define BEEPER_PIN           PE4
+
+// RX and TX on "RX" connector, RX on "HD-VTX" connector
+#define UART1_TX_PIN         PA9
+#define UART1_RX_PIN         PA10
+
+// on mezzanine connector
+#define UART2_TX_PIN         PD5
+#define UART2_RX_PIN         PD6
+
+// RX and TX on "HD-VTX" connector
+#define UART3_TX_PIN         PD8
+#define UART3_RX_PIN         PD9
+
+// for ESC TLM, with series 10k resistor
+#define UART5_TX_PIN         NONE
+#define UART5_RX_PIN         PB13 // TX signal pin-swapped
+
+// on mezzanine connector (other useful signals on these pins too)
+#define UART4_TX_PIN         PB9
+#define UART4_RX_PIN         PB8
+
+#define UART6_TX_PIN         NONE
+#define UART6_RX_PIN         NONE
+
+#define UART7_TX_PIN         NONE
+#define UART7_RX_PIN         NONE
+
+// on GPS connector
+#define UART8_TX_PIN         PE1
+#define UART8_RX_PIN         PE0
+
+#define UART9_TX_PIN         NONE
+#define UART9_RX_PIN         NONE
+
+#define UART10_TX_PIN        NONE
+#define UART10_RX_PIN        NONE
+
+// 8Mbps CAN transceiver installed with CAN termination, CANL/CANH available on GPS, mezzanine and servo connectors.
+#define FDCAN1_RX            PD0
+#define FDCAN2_RX            PD1
+
+#define LED0_PIN             PE5
+#define LED1_PIN             PE6
+#define LED2_PIN             NONE
+
+#define SPEKTRUM_RX_BIND_PIN NONE
+#define RX_BIND_PLUG_PIN     NONE
+
+#define RX_SPI_CS_PIN        PA15
+//#define RX_SPI_EXTI_PIN      PD4
+#define RX_SPI_BIND_PIN      NONE
+#define RX_SPI_LED_PIN       NONE
+#define RX_SPI_EXPRESSLRS_RESET_PIN PD10
+#define RX_SPI_EXPRESSLRS_BUSY_PIN PC7
+
+#define GYRO_1_EXTI_PIN      PE3
+#define GYRO_2_EXTI_PIN      PC15 // on mezzanine connector
+#define GYRO_1_CS_PIN        PB12
+#define GYRO_2_CS_PIN        PA15 // on mezzanine connector
+
+#define MOTOR1_PIN           PA3
+#define MOTOR2_PIN           PA2
+#define MOTOR3_PIN           PA1
+#define MOTOR4_PIN           PA0
+#define MOTOR5_PIN           PB1
+#define MOTOR6_PIN           PB0
+#define MOTOR7_PIN           PA7
+#define MOTOR8_PIN           PA6
+#define LED_STRIP_PIN        PC6
+
+// Index, Pin, 1-based occurrence of pin in fullTimerHardware, dma opt (use -1 for no DMA)
+// See timerio.c and timer_stm32h7xx.c
+
+#define TIMER_PIN_MAPPING \
+    TIMER_PIN_MAP( 0, PA3 , 2,  7 ) \
+    TIMER_PIN_MAP( 1, PA2 , 2,  6 ) \
+    TIMER_PIN_MAP( 2, PA1 , 2,  5 ) \
+    TIMER_PIN_MAP( 3, PA0 , 2,  4 ) \
+    TIMER_PIN_MAP( 4, PB1 , 2,  1 ) \
+    TIMER_PIN_MAP( 5, PB0 , 2,  0 ) \
+    TIMER_PIN_MAP( 6, PA7 , 2,  3 ) \
+    TIMER_PIN_MAP( 7, PA6 , 1,  2 ) \
+    TIMER_PIN_MAP( 8, PC6 , 2,  8 )
+
+#define DEFAULT_BLACKBOX_DEVICE     BLACKBOX_DEVICE_SDCARD
+
+#define DEFAULT_CURRENT_METER_SOURCE CURRENT_METER_ADC
+#define DEFAULT_VOLTAGE_METER_SOURCE VOLTAGE_METER_ADC
+
+#define USE_SPI_GYRO
```

Comment:
Already defined in target.h

Reply:
I'm not quite sure what file you're referring to, I don't have a `target.h` file here.  All the SPRacingH7 FC's have `#define USE_SPI_GYRO` defined, but I do see the following block in `common_post.h`:

```
#if defined(USE_GYRO_SPI_ICM20689) || defined(USE_GYRO_SPI_MPU6000) || defined(USE_GYRO_SPI_MPU6500) || defined(USE_GYRO_SPI_MPU9250) || defined(USE_GYRO_L3GD20) || defined(USE_GYRO_SPI_ICM42605) || defined(USE_GYRO_SPI_ICM42688P) || defined(USE_ACCGYRO_BMI160) || defined(USE_ACCGYRO_BMI270)
#ifndef USE_SPI_GYRO
#define USE_SPI_GYRO
#endif
#endif
```

Are you suggesting that `USE_SPI_GYRO` is no-longer required in all target's `config.h` files?

If so I can remove it from this PR, or, preferrably, leave it in this file (to be consistent), and then remove it from /all/ the remaining targets that still have it as a follow-up PR.

See below:

![image](https://github.com/betaflight/config/assets/57075/9ec1d823-aaf2-41fe-91ac-80bcc4495fa7)

Reply:
I'll add a second commit to this PR then.

Reply:
done.